File: tbsmith/php.ssl1304.com 4/models/DBConnector.php
<?php

class DBConnector {

    public $db;

    function __construct() {
        $host = '127.0.0.1';
        $user = 'root';
        $pass = 'root';
        $port = '3306';
        $dbname = 'fakeUsers';
        $this->db = new PDO("mysql:host=$host;port=$port;dbname=$dbname", $user, $pass);
        return $this->db;
    }

    public function getRandomUser() {
        $stmnt = $this->db->query("SELECT * from address ORDER BY rand() LIMIT 1");
        return $stmnt->fetchAll(PDO::FETCH_ASSOC);
    }

    public function addUser($firstname='', $lastname='', $address='', $city='', 
        $state='', $zip='', $phone='') {
            $stmnt = $this->db->prepare("INSERT INTO address(firstname, lastname,
                address, city, state, zip, phone) values (:fn, :ln, :add,
                :city, :st, :zip, :phone)");
            $stmnt->execute(array(
                ':fn'=>$firstname, 
                ':ln'=>$lastname,
                ':add'=>$address,
                ':city'=>$city,
                ':st'=>$state,
                ':zip'=>$zip,
                ':phone'=>$phone));
    }

}

?>

File: Jobies/jobies/models/DBConnector.py
#!/usr/bin/python

import mysql.connector


class DBConnector():

    def getConnection(self):
        self.cnx = mysql.connector.connect(host="127.0.0.1",
                                           port=8889,
                                           user="root",
                                           passwd="root",
                                           db="JobiesDB")
        return self.cnx



    # def updateUser(self, firstName='', lastName='', address='', city='', state='', phone='', zip=''):

    #   self.getConnection()

    #   sql = "UPDATE address \
    #          SET firstname=%(firstName)s, lastname=%(lastName)s, address=%(address)s, \
    #           city=%(city)s, state=%(state)s, zip=%(zip)s, phone=%(phone)s \
    #          WHERE id = 1182392"

    #   user_info = {
    #     'firstName': firstName,
    #     'lastName': lastName,
    #     'address': address,
    #     'city': city,
    #     'state': state,
    #     'zip': zip,
    #     'phone': phone
    #   }

    #   cursor = self.db.cursor()
    #   cursor.execute(sql, user_info)
    #   self.db.commit()
    #   cursor.close()
    #   self.db.close()    

    # def deleteUser(self, id=''):

    #   self.getConnection()

    #   sql = "DELETE FROM address WHERE id = 1182392"

    #   cursor = self.db.cursor()
    #   cursor.execute(sql)
    #   self.db.commit()
    #   cursor.close()
    #   self.db.close()
